client/.eslintrc

```diff
@@ -0,0 +1,13 @@
+{
+  "env": {
+    "browser": true,
+    "node": false
+  },
+  "rules": {
+    "globals": {
+      "define": false,
+      "require": false
+    },
```

Comment:
Do we need these globals, or should we be setting an `env` of "amd" or something?

> `amd` - defines `require()` and `define()` as global variables as per the [amd](https://github.com/amdjs/amdjs-api/wiki/AMD) spec.
> &mdash; via http://eslint.org/docs/user-guide/configuring.html#specifying-environments

Also, I think `globals` can be at the same level as `rules` and `env`, not nested: http://eslint.org/docs/user-guide/configuring.html#specifying-globals